### DISCUSSION TITLE
Vercel is asking for payment card to deploy Bit Bucket repository project even on Hobby plan.

### DISCUSSION THREAD

**Discussion by saad-devx** [vercel/community#4054] [Feature Question] [answered]
### Question

![image](https://github.com/vercel/community/assets/117750943/9d322df7-3c3e-4b99-920d-9108d4b8cbf0)
Vercel is asking to add a payment card even though i am on hobby plan upon importing a Bit Bucket repository. I just want to import a Bit Bucket repository from my workspace and deploy it on hobby plan of vercel. What is it that i am missing. I tried to find this issue everywhere including vercel docs but didn't got anything. Please help urgently.

**Comment by swarnava** [comment, marked as answer]
Hi,

You cannot import repository that part of a organization / workspace. In Hobby account, you can only import your Personal Git repository.

**Reply by saad-devx** [reply]
Thanks, i really appreciate it. Please explain one more thing i am new to this. How can i make a personal repository in my Bit Bucket account? Although it was 'private' but i was still prompted to add a payment card.
Let me provide more detail. Someone gave me creator access to his Bitbucket and there is a workspace with project having a repository all labeled as 'private'. And upon importing that we still getting prompted for payment. So do you mean that only owner should have access to Bitbucket not even a collaborator?

**Reply by swarnava** [reply]
Hi,

You can fork it and make it under own repo something like https://bitbucket.org/Dark-Reaper6/yourrepo which will allow you to import in Hobby plan.

>  Someone gave me creator access to his Bitbucket and there is a workspace with project having a repository all labeled as 'private'. 

This means these repo is part of an organization and you need a team account (with valid payment mode) to deploy workspace repo.

> So do you mean that only owner should have access to Bitbucket not even a collaborator?

Collaborator can also access repo and deploy at Vercel. But they need a team which is requires paid Vercel account.

**Reply by saad-devx** [reply]
Thank you so much!
